templates/server/src/main/resources/archetype-resources/core/src/main/java/__packageInPathFormat__/general/dataaccess/api/ApplicationPersistenceEntity.java

```diff
@@ -37,7 +37,8 @@ public ApplicationPersistenceEntity() {
 
   @Override
   @Id
-  @GeneratedValue(strategy = GenerationType.AUTO)
+  @GeneratedValue(strategy= GenerationType.AUTO,generator="native")
+  @GenericGenerator( name = "native", strategy = "native")
```

Comment:
here we could do such a change but with a switch via velocity depending on `dbType`. E.g.
```
#if ($dbType == 'mysql')
  @GeneratedValue(strategy= GenerationType.AUTO, generator="native")
  @org.hibernate.annotations.GenericGenerator( name = "native", strategy = "native")
#else
  @GeneratedValue(strategy= GenerationType.AUTO)
#end
```
Then also remove the dead import or if you do not want the annotation to be fully qualified also surround the import with an velocity if.